frontend/src/tests/ForestClientInput.spec.ts

```diff
@@ -102,4 +121,286 @@ describe('ForestClientInput', () => {
         // the forest client input should now be empty
         expect(inputField.value).toBe('');
     });
+
+    it('should raise error for forest client number not found', async () => {
+        const input = wrapper.find('#forestClientInput');
+        const inputField: HTMLInputElement = input.element as HTMLInputElement;
```

Comment:
I wonder if we could set the input and inputField to the beforeEach, we use them almost in all the test cases

Reply:
Ok, makes sense. I'll update it. Thanks.